docs/reference/formulas/smartwatts.md

```diff
@@ -36,33 +33,46 @@ You can use the following command to install SmartWatts:
 
 For running the SmartWatts Formula you need: a Source and a Destination, a Sensor that provides `HWPCReports` and a configuration.
 
-### Source and Destination
-For running SmartWatts we are using MongoDB as Source and InfluxDB 2.X as Destination as dockers containers.
+### Input and Output
+As any Formula, SmartWatts needs both inputs and outputs.
+We can choose those among [this list](../database/sources_destinations.md#summary)
 
-To start a MongoDB instance via the command line
-
-```sh
-docker run -d --name mongo_source -p 27017:27017 mongo
+#### MongoDB as input
+MongoDB can be used as input for Reports.
+You can start a MongoDB instance via a Docker container by running:
 ```
-And a InfluxDB 2.X instance
+docker run -d --name mongo_output -p 27017:27017 mongo:latest
+```
+The different images can be found on the [Docker Hub](https://hub.docker.com/_/mongo/tags)
+
+#### InfluxDB as output
+On the other hand, we can use InfluxDB instance as output for our Power Reports
 
+You can start an InfluxDB instance, in version >= 2.0.0 via a Docker container by running:
 ```sh
 docker run -p 8086:8086 -v "/tmp/powerapi-influx/data:/var/lib/influxdb2" -v "/tmp/powerapi-influx/config:/etc/influxdb2" influxdb:2
 ```
+The different images can be found on the [Docker Hub](https://hub.docker.com/_/influxdb/tags?name=2)
+
```

Comment:
To say that it necessary to create a organization, bucket, token in order to use the example below...